data_pipeline_api/java_api/src/main/java/uk/ramp/config/ConfigReader.java

```diff
@@ -0,0 +1,32 @@
+package uk.ramp.config;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import uk.ramp.file.FileDirectoryNormaliser;
+import uk.ramp.reader.Reader;
+import uk.ramp.yaml.YamlReader;
+
+class ConfigReader implements Reader<ImmutableConfig> {
+  private static final String LOCATION = "config.yaml";
```

Comment:
not sure if we'd expect this to be constant or an arg to the model

Reply:
Right now the FileApi class takes the directory (which contains config.yaml) as an input parameter. Happy to move this up to the API level and have the config.yaml path passed in instead of the directory. Let me know

Reply:
I've changed this now so that the actual file path is passed in as an arg to the File API